src/background/telemetry/exception-telemetry-listener.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BaseActionPayload } from 'background/actions/action-payloads';
+import { TelemetryEventHandler } from 'background/telemetry/telemetry-event-handler';
+import { Logger } from 'common/logging/logger';
+import * as TelemetryEvents from '../../common/extension-telemetry-events';
+import { UnhandledExceptionTelemetryData } from '../../common/extension-telemetry-events';
+
+export class ExceptionTelemetryListener {
+    constructor(private readonly telemetryEventHandler: TelemetryEventHandler) {}
+
+    public initialize(
+        logger: Logger,
+        extWindow: Window = window,
+        extConsole: Console = console,
+    ): void {
+        const sendExceptionTelemetry = (message: string, stackTrace?: string): void => {
```

Comment:
I wonder if there are any patterns we can predict ahead of time that would be a good idea to sanitize preemptively. Off the top of my head, I'm wondering if we should:

* Impose a size cap on message/stackTrace past which we truncate it (I've seen error messages be 100kb long if they manage to embed an entire axe-core result blob, for example)
* Avoid sending messages that include anything that looks like a URL
* Avoid sending messages that include any of the property names we or axe-core use to represent css selectors and/or HTML snippets

Reply:
Makes sense to me. Can you expand on that last item- do you have any examples of property names we wouldn't want to send and how we can generate a full list?

Reply:
I've gone ahead and addressed all the feedback in my most recent commits, except the 3rd item in this list.

Reply:
I'd suggest starting with the following properties (it's okay if for it to be incomplete/best-effort): `html`, `target`, `url`, `path`, `snippet`, `selector`, `elementSelector`, `cssSelector`

Reply:
Cool, just added that, thanks!